### PR TITLE
Getting the random word flag up and running for #26

### PR DESCRIPTION
**Pull request by rainrivas**
So this works by saying `slng -R` and it will return the default number of results (3).
if you pass a phrase and -R, the phrase takes precedence, e.g. `slng -R gucci` gives you results for gucci

### CODE REVIEW

**Comment by rainrivas** [comment]
Just realized tests are failing because I used "let" for the options. I'll use something else

**Comment by jwu910** [comment]
I think the random flag should return 1 random word and the top 3 results. Thoughts? I'm currently getting 3 different random words.

**Comment by rainrivas** [comment]
@jwu910 the random API returns random WORDS not a single word and its definitions. I am glad you brought this up since I also think we should just return a single word. 
I will have to refactor the structure of the project to put things into functions and call them that way. I will fix this up once the other PR (auto-help) is merged

**Comment by jwu910** [comment]
We have a lot of duplicate code I think we can clean up. Gonna do some here

**Comment by jwu910** [comment]
@rainrivas I dont think I understand the commander use of program.random and we shouldnt be using program.on('--random'...) or something long these lines. 

Lets review and chat when you get back =p

File: index.js [hunk 1]
```diff
@@ -8,57 +8,88 @@ const consoleWidth = () => {
...
+      console.log(chalk.bold.cyan('Definition: ') + result.definition);
+      console.log(chalk.bold.cyan('Score: ') + (result.thumbs_up - result.thumbs_down));
+      console.log(chalk.bold.green('Ayys: ') + result.thumbs_up + ' | ' + chalk.bold.red('Nayys: ') + result.thumbs_down);
+      console.log('='.repeat(consoleWidth()));
+    }
+  });
+}
+
+const getDefinition = (word) => {
+  let options = {
```
Comment (rainrivas) [inline index.js]: convert this to const

File: index.js [hunk 2]
```diff
@@ -8,57 +8,88 @@ const consoleWidth = () => {
...
+    },
+    headers: {
+      'Cache-Control': 'no-cache',
+      Accept: 'application/json'
+    }
+  }
+
+  request(options, function (err, res, body) {
+    if (err) throw new Error(err);
+    let trimRes;
```
Comment (rainrivas) [inline index.js]: convert this to var

File: index.js [hunk 3]
```diff
@@ -65,15 +65,19 @@ program
...
-    console.log('    $ slng gucci');
-    console.log('    $ slng \'square up\'');
-    console.log('');
-  });
+  console.log('');
+  console.log('  Examples:');
+  console.log('    $ slng gucci');
+  console.log('    $ slng \'square up\'');
+  console.log('');
+});
```
Comment (jwu910) [inline index.js]: ~~What happened to spacing? Lol~~

**Comment by rainrivas** [comment]
@jwu910 lol looks like tabs were replaced with spaces

**Review by jwu910 — APPROVED** [review]
LGTM Lets get this merged

**Comment by jwu910** [comment]
Note: We could set up a prettierrc and editorconfig

**Comment by rainrivas** [comment]
>Note: We could set up a prettierrc and editorconfig

@jwu910 nahh haha. I don't think we really need it for this toy app

**Comment by jwu910** [comment]
TOY APP?! This is the future!

**Pull request merged**